localstack/dashboard/infra.py

```diff
@@ -31,93 +25,17 @@
-    env_vars.update({
-        'AWS_ACCESS_KEY_ID': os.environ.get('AWS_ACCESS_KEY_ID') or 'test',
-        'AWS_SECRET_ACCESS_KEY': os.environ.get('AWS_SECRET_ACCESS_KEY') or 'test',
-        'AWS_DEFAULT_REGION': DEFAULT_REGION or os.environ.get('AWS_DEFAULT_REGION'),
-        'PYTHONWARNINGS': 'ignore:Unverified HTTPS request'
-    })
-    tmp_file_path = new_tmp_file()
-    error = None
-    with open(tmp_file_path, 'w') as err_file:
-        try:
-            return run(cmd, cache_duration_secs=cache_duration_secs, env_vars=env_vars, stderr=err_file)
-        except Exception as e:
-            error = e
-    if error:
-        LOG.warning('Error running command: %s %s %s' % (cmd, error, load_file(tmp_file_path)))
-        raise error
-
-
-def run_aws_cmd(service, cmd_params, env=None, cache_duration_secs=None):
-    cmd = '%s %s' % (aws_cmd(service, env), cmd_params)
-    if not is_api_enabled(service):
-        return '{}'
-    return run_cached(cmd, cache_duration_secs=cache_duration_secs)
-
-
-def cmd_s3api(cmd_params, env):
-    return run_aws_cmd('s3api', cmd_params, env)
-
-
-def cmd_es(cmd_params, env):
-    return run_aws_cmd('es', cmd_params, env)
-
-
-def cmd_kinesis(cmd_params, env, cache_duration_secs=None):
-    return run_aws_cmd('kinesis', cmd_params, env,
-        cache_duration_secs=cache_duration_secs)
-
-
-def cmd_dynamodb(cmd_params, env):
-    return run_aws_cmd('dynamodb', cmd_params, env)
-
-
-def cmd_firehose(cmd_params, env):
-    return run_aws_cmd('firehose', cmd_params, env)
-
-
-def cmd_sqs(cmd_params, env):
-    return run_aws_cmd('sqs', cmd_params, env)
-
-
-def cmd_lambda(cmd_params, env, cache_duration_secs=None):
-    return run_aws_cmd('lambda', cmd_params, env,
-        cache_duration_secs=cache_duration_secs)
-
-
-def aws_cmd(service, env):
-    # TODO: use boto3 instead of running aws-cli commands here!
-
-    cmd = '{ test `which aws` || . .venv/bin/activate; }; aws'
-    endpoint_url = None
-    env = aws_stack.get_environment(env)
-    if aws_stack.is_local_env(env):
-        endpoint_url = aws_stack.get_local_service_url(service)
-    if endpoint_url:
-        if not is_port_open(endpoint_url):
-            raise socket.error()
-        if endpoint_url.startswith('https://'):
-            cmd += ' --no-verify-ssl'
-        cmd = '%s --endpoint-url="%s"' % (cmd, endpoint_url)
-    cmd = '%s %s' % (cmd, service)
-    return cmd
-
-
 def get_kinesis_streams(filter='.*', pool={}, env=None):
     if MOCK_OBJ:
         return []
     result = []
     try:
-        out = cmd_kinesis('list-streams', env)
+        kinesis_client = aws_stack.connect_to_service('kinesis')
+        out = kinesis_client.list_streams()
         out = json.loads(out)
         for name in out['StreamNames']:
             if re.match(filter, name):
-                details = cmd_kinesis('describe-stream --stream-name %s' % name, env=env)
+                details = kinesis_client.describe_stream('StreamArn=%s' % name)
```

Comment:
nit: I think we need to pass in the actual attributes here, instead of a string `'<attrName>=<value>'`
```
kinesis_client.describe_stream(StreamArn=name)
```